cmd/ocm/login/cmd.go

```diff
@@ -143,6 +146,13 @@ func init() {
 			"this option is provided then the user name and password will be stored "+
 			"persistently, in clear text, which is potentially unsafe.",
 	)
+	flags.BoolVar(
+		&args.oauthLogin,
+		"oauth-login",
```

Comment:
One final nitpick here. Let's use a flag name like `--use-auth-code`. Oauth is a set of standards as opposed to a single type of login.

This is similar to how azure `az` operates as well https://learn.microsoft.com/en-us/cli/azure/authenticate-azure-cli-interactively

Reply:
Ack. Changed to `--use-auth-code`